Automattic/delphin
# Setup flow: better explain next steps on success screen

Implements #1094:

> After completing the domain setup flow on get.blog, further action is going to be needed on the connected host. 
> 
> For example, if you choose to start a new WordPress.com blog, you will end up on this screen:
> 
> ![ranimac3 screenshot 2016-12-21 at 11 44 01](https://cloud.githubusercontent.com/assets/426518/21384336/d2ee1f70-c772-11e6-913b-22bc5bd3cae4.png)
> 
> We say that setup is "complete" and the domain is "ready", but in fact nothing is complete or ready yet, and you have to continue to WordPress.com to actually create the blog. If you don't click the "Create my WordPress.com blog" button on this screen, there is no other way to resume the setup process (other than resetting to default settings and restarting the setup process).
> 
> Let's change the copy on this screen to make the whole thing clearer:
> 
> > Setup: proceed to WordPress.com.
> > 
> > Your domain is almost ready, now it's time to start building your new blog. It will be available at example.blog.
> >
> > Start my WordPress.com blog

### After changes

#### New blog

##### Pressable
![screen shot 2016-12-22 at 09 29 58](https://cloud.githubusercontent.com/assets/699132/21420885/6934e83e-c830-11e6-882c-ca53696a1bab.png)

##### WordPress.com
![screen shot 2016-12-22 at 09 27 49](https://cloud.githubusercontent.com/assets/699132/21420887/69354068-c830-11e6-82ea-589026f37eb2.png)

#### Existing blog

##### Pressable
![screen shot 2016-12-22 at 10 03 20](https://cloud.githubusercontent.com/assets/699132/21420888/69355eea-c830-11e6-88ac-c70b3a14471f.png)

##### WordPress.com
![screen shot 2016-12-22 at 10 01 47](https://cloud.githubusercontent.com/assets/699132/21420886/6934e140-c830-11e6-907b-ec0c76f23577.png)

### Testing
1. Open http://delphin.localhost:1337 or https://delphin.live/?branch=update/wording-success-screen.
2. Go to My Domains. If you don't have any domains, go a purchase one.
3. Go through Set Up flow and try all combinations.
4. New WordPress.com blog.
5. New Pressable blog.
6. Existing WordPress.com blog (you can use gziolotest54321.wordpress.com).
7. Existing Pressable blog (you can use scruffiantest.mystagingwebsite.com).
8. Make sure all success screens look like on the screens provided above.

### Review


@Automattic/sdev-feed
